File: php/src/SSO/Request.php
<?php

namespace SSO;

class Request
{
    /**
     * @var string
     */
    private $name;

    /**
     * @var SSOToken
     */
    private $token;

    /**
     * @param string $name
     * @param SSOToken $token
     */
    public function __construct($name, SSOToken $token)
    {
        $this->name = $name;
        $this->token = $token;
    }

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @return SSOToken
     */
    public function getToken()
    {
        return $this->token;
    }
}

File: CSharp/SingleSignOnKata/sso/Request.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SingleSignOnKata.sso
{
    public class Request
    {
        private String name;
        private SSOToken token;

        public Request(String name, SSOToken token)
        {
            this.name = name;
            this.token = token;
        }

        public SSOToken getSSOToken()
        {
            return this.token;
        }

        public String getName()
        {
            return this.name;
        }
    }
}
